ui/src/kapacitor/apis/index.js

```diff
@@ -100,3 +100,18 @@ export const updateTask = async (
     throw error
   }
 }
+
+export const getLogStream = kapacitor =>
+  fetch(`${kapacitor.links.proxy}?path=/kapacitor/v1preview/logs`, {
+    method: 'GET',
+    headers: {'Content-Type': 'application/json'},
+  })
+
+export const getLogStreamByRuleID = (kapacitor, ruleID) =>
+  fetch(
+    `${kapacitor.links.proxy}?path=/kapacitor/v1preview/logs?task=${ruleID}`,
```

Comment:
Will this have to be changed to `/v1` (no preview)?

Reply:
☝️

Reply:
@121watts in the short term let's keep this path.  In the longer term, we'll add this link to the kapacitor rest resource (https://github.com/influxdata/chronograf/issues/2415)

Reply:
To add a bit of color here. When the API for a feature is likely to change, or if we're uncertain if it will change, we release it under `/v1preview` in kapacitor. After it's been stable for a release or two, we change it to `/v1`.